app/src/test/java/ch/hikemate/app/model/route/saved/SavedHikesRepositoryFirebaseTest.kt

```diff
@@ -0,0 +1,63 @@
+package ch.hikemate.app.model.route.saved
+
+import kotlin.time.Duration.Companion.seconds
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.UnconfinedTestDispatcher
+import kotlinx.coroutines.test.runTest
+import kotlinx.coroutines.test.setMain
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.Test
+
+class SavedHikesRepositoryFirebaseTest {
+  private lateinit var savedHikesRepositoryFirebase: SavedHikesRepositoryFirebase
+
+  @OptIn(ExperimentalCoroutinesApi::class)
+  @Before
+  fun setUp() {
+    Dispatchers.setMain(UnconfinedTestDispatcher())
+
+    savedHikesRepositoryFirebase = SavedHikesRepositoryFirebase()
+  }
+
+  @Test
+  fun loadSavedHikesReturnsDummyData() =
+      runTest(timeout = 5.seconds) {
+        // When
+        val savedHikes = savedHikesRepositoryFirebase.loadSavedHikes()
+
+        // Then
+        assertEquals(3, savedHikes.size)
+        assertEquals("1", savedHikes[0].id)
+        assertEquals("Hike 1", savedHikes[0].name)
+        assertEquals("2", savedHikes[1].id)
+        assertEquals("Hike 2", savedHikes[1].name)
+        assertEquals("3", savedHikes[2].id)
+        assertEquals("Hike 3", savedHikes[2].name)
+      }
+
+  @Test
+  fun addSavedHikeDoesNothing() =
+      runTest(timeout = 5.seconds) {
+        // Add saved hike is a placeholder for now, it does nothing under the hood
+
+        // When
+        savedHikesRepositoryFirebase.addSavedHike(SavedHike("4", "Hike 4", null))
+
+        // Then
+        assert(true)
+      }
+
+  @Test
+  fun removeSavedHikeDoesNothing() =
+      runTest(timeout = 5.seconds) {
+        // Remove saved hike is a placeholder for now, it does nothing under the hood
+
+        // When
+        savedHikesRepositoryFirebase.removeSavedHike(SavedHike("4", "Hike 4", null))
+
+        // Then
+        assert(true)
```

Comment:
Nitpick: Couldn't you test something instead of asserting true? Like if the number of hikes is the same before and after the operation. Even if this isn't the expected behavior of a repository, it's the expected behavior of a dummy repository. (It then requires the file to be renamed to make sense and not confuse people)

Reply:
Indeed it makes sense to have at least something going on under the hood. I added a mutable list and tests for the new (exciting) features of the dummy repository in [90e48b4c](https://github.com/HikeMate/hikeMateApp/pull/66/commits/90e48b4c0e6f6cbc37451c63160555d35e679da2).

Reply:
Lgtm! Thanks!